src/HTTP.jl

```diff
@@ -322,6 +322,9 @@ function ServerWS(h::Function, w::Function; kwargs...)
     ServerWS(RequestHandlerFunction(h), WSHandlerFunction(w); kwargs...)
 end
 
+to_IPAddr(ip::IPAddr) = ip
```

Comment:
I suppose this is good style, considering. Alternatively, we could add
```
   IPv4(i::IPv4) = ip
```
but risk that Sockets add a method like that later, and we'd end up with warnings. What do you think?

We need a test for this. But there's also other new methods that are not posted. Will have a look at current test coverage and probably submit a PR to get back up to ~100% coverage.

Reply:
I've followed HTTP convention (which is in line with Julia Base) to lowercase `IPAddr` in a function name (so now it's `to_ipaddr`) and have added tests for it. Adding a `IPv4(::IPv4)` method here would be type piracy: something like that should be added in the Sockets stdlib directly.